test/unit/annotation_layer_spec.js

```diff
@@ -581,6 +581,97 @@ describe('Annotation layer', function() {
     });
   });
 
+  describe('CheckboxWidgetAnnotation', function() {
+    var checkboxWidgetDict;
+
+    beforeEach(function (done) {
+      checkboxWidgetDict = new Dict();
+      checkboxWidgetDict.set('Type', Name.get('Annot'));
+      checkboxWidgetDict.set('Subtype', Name.get('Widget'));
+      checkboxWidgetDict.set('FT', Name.get('Btn'));
+
+      done();
+    });
+
+    afterEach(function () {
+      checkboxWidgetDict = null;
+    });
+
+    it('should have proper flags',
+        function() {
+      var checkboxWidgetRef = new Ref(124, 0);
+      var xref = new XRefMock([
+        { ref: checkboxWidgetRef, data: checkboxWidgetDict, }
+      ]);
+
+      var checkboxWidgetAnnotation =
+          annotationFactory.create(xref, checkboxWidgetRef);
```

Comment:
For all occurrences in this file: remove two spaces from the indentation here. We indent everything with two spaces.